src/transformers/models/conditional_detr/image_processing_conditional_detr.py

```diff
@@ -935,10 +935,21 @@ def resize_annotation(
 
     # Copied from transformers.models.detr.image_processing_detr.DetrImageProcessor.rescale
     def rescale(
```

Comment:
Some of the image processors use this implementation of `rescale` instead of the one in `BaseImageProcessor`. This is because they use the keyword arguments `rescale_factor` instead of `scale`. This is just a mistake I made when porting form feature extractors to image processors. This argument will be deprecated in place of scale in a follow-up PR